File: 070_Climbing_Stairs.py
class Solution:
    def climbStairs(self, n):
        """
        :type n: int
        :rtype: int
        """
        f0, f1 = 1, 1
        for i in range(2, n + 1):
            t = f1
            f1 = f0 + f1
            f0 = t
        return f1

File: 070_Climbing_Stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
        int f0 = 1, f1 = 1;
        for (int i = 2; i <= n; ++i) {
            int temp = f1;
            f1 = f0 + f1;
            f0 = temp;
        }
        return f1;
    }
};